### ISSUE TITLE
Reverse edge direction between Program/Project/Case/Sample/Aliquot

### ISSUE THREAD

**Issue by adamstruck** [bmeg/bmeg-etl#318]
Or we may want to have bidirectional edges in this section of the graph for convenience.

**Comment by kellrott** [comment]
Draft for new edge schema:
```
Program	hasProject	Project
Project	hasCase	Case
Case	hasSample	Sample
Sample	hasAliquot	Aliquot
Case	hasTreatment	Compound
Aliquot	hasPhenotype	Phenotype
Aliquot	hasCallset	Callset
Aliquot	hasGeneExpression	GeneExpression
Aliquot	hasCNA	CopyNumberAlteration
Aliquot	hasTranscriptExpression	TranscriptExpression
Callset	hasSomaticMutation	Allele
Allele	alleleIn	Gene
PFAMClan	hasMember	PFAMFamily
PFAMFamily	hasAlignment	Protein
Gene	hasTranscript	Transcript
Transcript	hasExon	Exon
Transcript	hasProtein	Protein
Protein	hasStructure	ProteinStructure
MinimalAllele	alleleIn	Gene
G2PAssociation	hasMinimalAlleleFeature	MinimalAllele
G2PAssociation	hasGeneFeature	Gene
G2PAssociation	hasSupportingReference	Publication
G2PAssociation	hasEnvironment	Compound
G2PAssociation	hasPhenotype	Phenotype
G2PAssociation	hasAlleleFeature	Allele
DrugResponse	responseTo	Compound
DrugResponse	responseIn	Aliquot
GeneOntologyTerm	isA	GeneOntologyTerm
Gene	hasAnnotation	GeneOntologyTerm
PFAMFamily	hasAnnotation	GeneOntologyTerm
```

**Comment by adamstruck** [comment]
MinimalAllele was changed to GenomicFeature in ec36533

```
GenomicFeature	featureIn	Gene
G2PAssociation	hasGenomicFeature	GenomicFeature
```

**Comment by kellrott** [comment]
Addressed in #332
